### PR TITLE
TF Migration 1.x >> TF 2.x (d_traineval.ipynb) Notebook Fix

### PR DESCRIPTION
**Pull request by kalyanipande-tudip**
- Updated all the instructions in the d_traineval.ipynb notebook according to TensorFlow 2.X requirements.
- Added the command to check the version of Tensorflow installed in the target and solution notebook.
- Added the command to change the ownership of repo to user in the target and solution notebook.
- Related Buganizer: http://b/157448709

### CODE REVIEW

File: courses/machine_learning/deepdive/03_tensorflow/d_traineval.ipynb [hunk 1]
```diff
@@ -9,12 +9,30 @@
    "source": [
```
Comment (gstripling) [inline courses/machine_learning/deepdive/03_tensorflow/d_traineval.ipynb:9]: <p>I noticed that the reference to TensorBoard has been removed in the introductory paragraph, e.g. "We also use TensorBoard to monitor the training."  If we are to remove this reference, then we should also remove the heading "Monitor training" and aslo remove the cell that initializes the    OUTDIR = './taxi_trained'    variable, as that is what TensorBoard would use to build the graph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: courses/machine_learning/deepdive/03_tensorflow/labs/d_traineval.ipynb [hunk 2]
```diff
@@ -9,12 +9,30 @@
    "source": [
```
Comment (gstripling) [inline courses/machine_learning/deepdive/03_tensorflow/labs/d_traineval.ipynb:9]: <p>Same comment regarding the Monitor Training.  If TensorBoard no longer being used, no need for the "Monitor training" section or the code:  OUTDIR = './taxi_trained'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (priumoraes) [reply courses/machine_learning/deepdive/03_tensorflow/labs/d_traineval.ipynb:9]: I agree with Gwendolyn's comment.

**Comment by kalyanipande-tudip** [comment]
Hi @priumoraes / @gstripling 

I agree with your suggestions but we can not completely remove the code: OUTDIR = './taxi_trained' here as this is further used in running and evaluating the model in the next section of notebook.

Hence I have updated the instructions accordingly. Please let me know if we can move ahead with this
change.

-Thanks

**Comment by gstripling** [comment]
Hi Kalyani - good catch.  I noticed that you also removed the heading "Monitor training".  Thanks.

**Pull request merged**